docs/source/markdown/podman-pod-create.1.md

```diff
@@ -96,6 +96,10 @@ Set network mode for the pod. Supported values are
   - **port_handler=rootlesskit**: Use rootlesskit for port forwarding. Default.
   - **port_handler=slirp4netns**: Use the slirp4netns port forwarding.
 
+#### **--network-alias**=strings
+
+Add network-scoped alias for the container
+
```

Comment:
Ditto (re: needing expansion)

Reply:
"Add a DNS alias for the container. When the container is joined to a CNI network with support for the dnsname plugin, the container will be accessible through this name from other containers in the network.

Reply:
Updated and pushed, thanks.